Networking/Repository/RequestExecutor.swift

```diff
@@ -41,53 +35,47 @@ public protocol RequestExecutorType {
     otherwise fails, and returns the response.
  */
 internal final class RequestExecutor: RequestExecutorType {
-    
     private let _sessionManager: Alamofire.SessionManager
+    private let _encoding: Encoding
     
-    internal init(sessionManager: Alamofire.SessionManager) {
+    internal init(sessionManager: Alamofire.SessionManager, encoding: Encoding) {
         _sessionManager = sessionManager
+        _encoding = encoding
     }
     
-    func perform(
-        method: NetworkingMethod,
-        url: URL,
-        parameters: [String: Any]? = .none,
-        headers: [String: String]? = .none) -> HTTPResponseProducer {
-            return _sessionManager
-                .request(url,
-                         method: method.toHTTPMethod(),
-                         parameters: parameters,
-                         encoding: Encoding(),
-                         headers: headers)
+    func perform(method: NetworkingMethod, url: URL, parameters: [String: Any]? = .none, headers: [String: String]? = .none) -> HTTPResponseProducer {
```

Comment:
maybe we can add the enconding as a parameter with _enconding as the default value? so you don't have to create a new rerpository if you need a specific enconding for a special request

Reply:
mmm good idea

Reply:
@DiegoQuiros94 I'll add this as a parameter but in another PR I'll improve it so you do not need to provide a full encoding object when you only want to change the encoding of that particular request (also because that overlaps with another card)

Reply:
sounds good